frontend/components/Dashboard/Profile/ProfileCard.tsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react'
+import { useTranslation } from '../../../config/i18n'
+import FacebookIcon from '../../../components/Icons/FacebookIcon'
+import InstagramIcon from '../../../components/Icons/InstagramIcon'
+import YoutubeIcon from '../../../components/Icons/YoutubeIcon'
+import GlobeIcon from '../../../components/Icons/GlobeIcon'
+import ExternalLink from '../../../elements/ExternalLink'
+import theme from '../../../theme'
+
+const ProfileCard: React.FC = () => {
+  const { t } = useTranslation('profile')
+
+  const user = {
+    name: 'Avocado 🥑 King 🤴🏼',
+    handle: 'avocado-me',
+    image: '/images/robin-small.png',
+    speaks: ['English'],
+    learns: ['Japanese, French'],
+    likes: ['cooking, reading, movies, design'],
+    bio:
+      'Praesent commodo a quis at dui taciti sagittis senectus inceptos nascetur, dictumst accumsan quam tortor dictum in ultrices natoque sodales, venenatis et iaculis aliquet blandit mi mauris faucibus molestie. Libero suspendisse urna placerat elit non est metus vivamus justo, duis nam ridiculus mattis eu gravida tellus curae, maecenas nisi pellentesque elementum imperdiet mus ac varius.',
+    location: 'San Francisco, United States',
+    facebook: 'https://www.facebook.com/robinmacpherson.co',
+    instagram: 'https://instagram.com/my-link',
+    youtube: 'https://www.youtube.com/user/TheLifeOfRob/',
+    website: 'https://robinmacpherson.com',
+  }
+
+  const name = user.name || user.handle
+  const showSeparator = user.facebook || user.instagram || user.youtube || user.website
+
+  return (
+    <div className="profile-card">
+      <div className="profile-header">
+        <h1 className="profile-name">{name}</h1>
+
```

Comment:
Is there any reason for the empty lines here and there throughout the file amongst the markup?

Reply:
I almost always add empty lines between sibling elements unless it's very similar and repeated (like the `<p>` tags below and `social-link`s). It was something we did when I was at Topcoder and I've just continued doing it

Reply:
Hmm I see, I feel like this might be the kind of thing that would be better either as a convention or not. It's definitely not like a huge thing of course but if each person has a very particular style preference then we end up with lots of files looking quite different from each other depending on who wrote them and at SS you'll often get people who end up submitting PRs where they have an auto-formatter set up one way and it changes a bunch of stuff but then the next person comes along and re-formats it their way 😂 I feel like readability of a codebase is very important and consistency is important there

Reply:
Generally it's best to just use an eslint rule for this kind of thing so that it's just all the same, but I just tried to find an eslint rule for empty lines in HTML and I couldn't find one. Prettier also doesn't change empty lines on HTML files, so we shouldn't run into a situation where someone saves an HTML file and it auto-formats. I personally don't think we should have a rule around empty lines in HTML, no empty lines will make some files with dense information look really confusing, and enforcing empty lines for everything can make files bloated and lengthy. I think that's also why there isn't an eslint rule for this specifically, because it's an "it depends" situation on when to use empty lines for readability. In terms of a convention, my personal preference is just, "format it so it's readable"